bin/ovpn_genconfig

```diff
@@ -104,6 +110,11 @@ while getopts ":r:s:du:cp:DNm:" opt; do
     esac
 done
 
+# if new routes were not defined with -r, use default
+[ ${#TMP_ROUTES[@]} -gt 0 ] && OVPN_ROUTES=("${TMP_ROUTES[@]}")
```

Comment:
Why do this if it happens on line 123/134 anyways?  I don't see what this fixes.